src/library/Pagination/Pages.js

```diff
@@ -0,0 +1,151 @@
+/* @flow */
```

Comment:
Some thoughts on focus management...

* If click previous and hit first page, button becomes disabled and focus is lost
* If click next and hit last page, button becomes disabled and focus is lost
* Should we manage focus in these scenarios and set focus somewhere else, like the opposite button?